### DISCUSSION TITLE
cursor stop moving for a split second when playing

### DISCUSSION THREAD

**Discussion by Theadripper** [ppy/osu#25214] [Q&A] [answered]
### Type

Game behaviour

### Bug description

it started like that after updating to newest version, it happend everytime on any beatmaps

### Screenshots or videos

https://github.com/ppy/osu/assets/83263748/2588b622-81b4-4395-85ac-c94eb9c7dc4a

### Version

2023.1008.1

### Logs

[database.log](https://github.com/ppy/osu/files/13060592/database.log)
[input.log](https://github.com/ppy/osu/files/13060593/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/13060594/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/13060595/network.log)
[performance.log](https://github.com/ppy/osu/files/13060596/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/13060597/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/13060598/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/13060599/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/13060600/runtime.log)
[updater.log](https://github.com/ppy/osu/files/13060601/updater.log)

**Comment by bdach** [comment]
Please take another video of this with frame graph shown (ctrl-f11 twice).

**Reply by Theadripper** [reply]
https://github.com/ppy/osu/assets/83263748/97fc7f72-a4bf-4218-be3d-6e78cffca78b

**Reply by bdach** [reply]
Something else on your system is hogging input causing the input thread to hang.

Do you have any external drivers enabled? If yes, try disabling them. Any high polling rate devices? Try decreasing the poll rate on them.

**Reply by Theadripper** [reply]
only my mouse is connected to my system and i tried to change the polling rate of my mouse from 1000 to 500 and it still happened, i dont have the same problem on osu! classic or any game. any other suggestions for how to fix it?

https://github.com/ppy/osu/assets/83263748/2e600a40-4f87-453c-bbce-a667c4e54529

**Reply by bdach** [reply]
No idea sorry. I'd probably start killing background applications and seeing if anything helps.

This is a somewhat frequent complaint but more other than not it turns out something else is interfering with lazer reading input. I guess if you don't find anything you could try switching to single thread mode as it will reduce the input thread fps but that's a nuclear solution of sorts.

**Reply by Theadripper** [reply]
i've restarted my device and killing all my bg apps and its still chug, then i tried switching to single thread mode and it just make the whole game stutter, kinda better cause the lag not as long as multithread mode, guess it's time to go back to osu! classic ಠ⁠﹏⁠ಠ  thank you for your help and i hope it can be fixed......

https://github.com/ppy/osu/assets/83263748/dbe14903-c718-41bf-9f23-9f63d469a006

**Reply by bdach** [reply]
> i hope it can be fixed

It will not be fixed if we don't know what's wrong. I assure you this does not happen on any device or OS I test with.

**Reply by Theadripper** [reply, marked as answer]
finaly found the problem, so i had my wallpaper to change after 1 minute and that change the theme color of the whole system even on fullscreen (idk why) , so i just set my wallpaper to static and it fixed the problem.... ლ⁠(⁠^⁠o⁠^⁠ლ⁠)
sorry for the inconvenience, that was totally my problem
![image](https://github.com/ppy/osu/assets/83263748/39b41b2f-8f15-4315-b483-045c82e28711)

https://github.com/ppy/osu/assets/83263748/9aa8b4a9-7544-456c-9c44-ffa49b014f15
